src/Extensions/Abstractions/Activities/TimedScope.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.Omex.Extensions.Abstractions.Activities
+{
+	/// <summary>
+	/// Logs duration of activity
+	/// </summary>
+	/// <remarks>This class will be replaced with Activity after move to net 5, since it will be trackable and disposible</remarks>
```

Comment:
nit:
```suggestion
	/// <remarks>This class will be replaced with Activity after move to net 5, since it will be trackable and disposable</remarks>
```